src/core-services/account/schemas/account.graphql

```diff
@@ -349,6 +362,14 @@ type SetAccountProfileLanguagePayload {
   clientMutationId: String
 }
 
+"The response from the `setUserPermissionsMutation` mutation"
+type SetUserPermissionPayload {
```

Comment:
Right now the return of this corresponding resolver does not match this type.

Reply:
@rosshadden ... I've updated to fix this .. done